grammars/rescript.tmLanguage.json

```
@@ -127,7 +127,7 @@
         {
           "name": "string.template",
           "begin": "([a-z_][0-9a-zA-Z_]*)?(`)",
-          "end": "`",
+          "end": "[^\\]`",
```

Should this be ```"[^\\\\]`"``` to json encode the escaped backslash.

---

why so many backslash? `\\` is already an escaped backslash, isn't it?

---

or is it because we need to add another layer of escaping because it's a json? Then yes!

---

yeah I guess this makes sense:
```suggestion
          "end": "[^\\\\]`",
```